tests/spec/w3c/headers-spec.js

```diff
@@ -998,27 +997,24 @@ describe("W3C — Headers", () => {
       Object.assign(ops.config, newProps);
       const doc = await makeRSDoc(ops);
       expect(
-        $(
-          ".head .copyright a[href='https://www.w3.org/community/about/agreements/fsa/']",
-          doc
+        doc.querySelectorAll(
+          ".head .copyright a[href='https://www.w3.org/community/about/agreements/fsa/']"
         ).length
       ).toEqual(1);
-      expect($(".head h2", doc).text()).toMatch(/Final Business Group Report/);
-      expect(
-        $("dt:contains('This version:')", doc)
-          .next("dd")
-          .text()
-      ).toMatch(/http:\/\/THIS/);
-      expect(
-        $("dt:contains('Latest published version:')", doc)
-          .next("dd")
-          .text()
-      ).toMatch(/http:\/\/LATEST/);
-      const $sotd = $("#sotd", doc);
-      expect($sotd.find("a[href='http://WG']").length).toEqual(1);
-      expect($sotd.find("a:contains(WGNAME)").length).toEqual(1);
+      expect(doc.querySelector(".head h2").textContent).toMatch(
+        /Final Business Group Report/
+      );
+      expect(doc.querySelectorAll("dd")[0].textContent).toMatch(
```

Comment:
Could we additionally make sure that the preceding `dt` contains `"This version":`?

Reply:
I just couldn't find any direct JS function, I thought the HTML structure wouldn't change. Is there a way to do that without manually iterating?
If not, I think I will make a util function for this file, as there are a couple more such cases in the file.

Reply:
Unfortunately, you might need to add the util function.

Reply:
We need to substitute for the `.next("...")` jquery function. It selects the next sibling which satisfies the selector passed. There are several `.next` occurences in the original `headers-spec.js` file. I remember doing something similar using `.nextElementSibling`, it didn't work out in some of them, as it could be any subsequent sibling, not necessarily the next. 

I think we do need the looping function.

Reply:
`.next("...")` is actually looser because it doesn't check `<dd>` and `<dt>` are direct neighbor siblings. My thought:

```js
const terms = doc.querySelectorAll("dt");
expect(terms[0].textContent).toContain("...");
expect(terms[0].nextElementSibling.localName).toBe("dd");
expect(terms[0].nextElementSibling.textContent).toContain("...");
```

Reply:
IMO `toMatch` should be replaced with `toContain` as we don't need regex here.

Reply:
Yep it works, previously I had used the same approach using ```.nextSibling()``` My bad :sweat: